Riot/Categories/UIScrollView.swift

```diff
@@ -30,5 +30,18 @@ extension UIScrollView {
             self.scrollRectToVisible(rect, animated: animated)
         }
     }
+
+    /// Scroll to bottom of the receiver.guard let self = self else { return }
```

Comment:
```suggestion
    /// Scroll to bottom of the receiver.
```

I guess this is not meant to be here :)

Reply:
🤦‍♂️ silly Xcode inserting that when i tap Tab.